tests/test_association_task.py

```diff
@@ -240,12 +341,15 @@ def _store_dia_objects_and_sources(self):
             scatter_arcsec=-1,)
         # Set the DIAObject fluxes and number of associated sources.
         for dia_object in dia_objects:
-            dia_object['nDiaSources'] = 2
             for filter_name in self.filter_names:
-                dia_object['psFluxMean_%s' % filter_name] = 1
-                dia_object['psFluxMeanErr_%s' % filter_name] = 1
-                dia_object['psFluxSigma_%s' % filter_name] = 1
-        assoc_db.store_dia_objects(dia_objects, True)
+                dia_object["pixelId"] = 225823
+                dia_object['%sPSFluxMean' % filter_name] = 1
+                dia_object['%sPSFluxMeanErr' % filter_name] = 1
+                dia_object['%sPSFluxSigma' % filter_name] = 1
+                dia_object['%sPSFluxNdata' % filter_name] = 1
+
+        dateTime = dafBase.DateTime(nsecs=1400000000 * 10**9 - 1000)
```

Comment:
```suggestion
        dateTime = dafBase.DateTime(nsecs=1400000000 * 1e9 - 1000)
```
This is kind of unusual notation anyway; why not just write out the number?

Reply:
Changed to a string input.